common/script/cron.js

```diff
@@ -28,7 +28,7 @@ function sanitizeOptions (o) {
   let dayStart = !_.isNaN(ref) && ref >= 0 && ref <= 24 ? ref : 0;
 
   let timezoneOffset;
-  let timezoneOffsetDefault = Number(moment().zone());
+  let timezoneOffsetDefault = Number(moment().utcOffset());
```

Comment:
let's not change these. In the deprecation warning it says that the new value can be different from the previous one